.github/deploy_docs.sh

```diff
@@ -5,65 +5,70 @@ if [[ $(git rev-parse --abbrev-ref HEAD) != "master" ]]; then
     exit 0
 fi
 
-# Assumes being called from the Biopython repository's root folder,
```

Comment:
I would retain comment lines 8 to 15 about this script being run after sphinx-build